packages/common/src/HardhatConfig.js

```diff
@@ -27,12 +28,32 @@ function getHardhatConfig(configOverrides, workingDir = "./") {
         },
       },
     },
+    ovm: {
+      solcVersion: solcVersion,
+    },
     networks: {
       hardhat: {
         gas: 11500000,
         blockGasLimit: 11500000,
         allowUnlimitedContractSize: false,
         timeout: 1800000,
+        testBlacklist: ["ovm"],
+      },
+      optimism: {
```

Comment:
Should this somehow specify that this is optimism running locally? This isn't the public optimism network, right?

Reply:
yeah its just local, for now I'm going to leave it as is because there aren't many dependencies. When we add in the public networks i'll make the change. WDYT?